### ISSUE TITLE
Paper missing from SIGMORPHON 2021 proceedings

### ISSUE THREAD

**Issue by tpimentelms** [acl-org/acl-anthology#1997]
## Option 2: Revision or erratum

[LiWilson_sigmorphon2021.pdf](https://github.com/acl-org/acl-anthology/files/9004991/LiWilson_sigmorphon2021.pdf)

- [x] I have attached the revised PDF or erratum to this issue
- [x] I have written a brief description of the changes below

It seems the attached paper (Li and Wilson, 2021) appears in the SIGMORPHON proceedings on github: https://sigmorphon.github.io/workshops/2021/2021_SIGMORPHON_Proceedings.pdf
It, however, does not appear in the acl anthology: https://aclanthology.org/volumes/2021.sigmorphon-1/

To avoid confusion, the same authors submitted two papers to the workshop: a Li & Wilson (system JHU-SFU), and a Wilson & Li (system JHU) paper.
The Wilson & Li appears in the anthology already, but the Li and Wilson (which is attached) does not.

Is there a way to fix this and add the attached paper to the anthology?

**Comment by xinru1414** [comment]
@tpimentelms the official proceedings we received and ingested ([here](https://aclanthology.org/2021.sigmorphon-1.0.pdf)) is different from  the one you linked above.  There are several papers in the github version missing from the official version. 

@mjpost Could this be a mistake on the workshop end? Or maybe these papers are not supposed to be published in this venue?

**Comment by tpimentelms** [comment]
Yeah, I think the first version of the proceedings was indeed missing papers. I think several of the missing papers are already uploaded to the ACL anthology, though. If I'm not mistaken, the only one still missing from the anthology is this Li and Wilson one.
